std_msgs/Float32 xCord
std_msgs/Float32 yCord
std_msgs/Bool rightCam
